causalpy/pymc_experiments.py

```diff
@@ -957,6 +957,198 @@ def summary(self) -> None:
         self.print_coefficients()
 
 
+class RegressionKink(ExperimentalDesign):
+    """
+    A class to analyse sharp regression kink experiments.
+
+    :param data:
+        A pandas dataframe
+    :param formula:
+        A statistical model formula
+    :param kink_point:
+        A scalar threshold value at which there is a change in the first derivative of
+        the assignment function
+    :param model:
+        A PyMC model
+    :param running_variable_name:
+        The name of the predictor variable that the kink_point is based upon
+    :param epsilon:
```

Comment:
I know you suggested not having the notebook as a place for explaining the theory of kink designs, but i feel like the differences between tweaking at least one of epsilon/bandwidth parameters could be mentioned or shown. 

It's fine if tweaking them isn't needed for your example, but it'd be good to hint at why they are there.

Reply:
I've added an example to demonstrate use of the `bandwidth` parameter. And I've added an admonition box to explain what epsilon does.